src/libraries/System.Net.Ping/tests/FunctionalTests/PingTest.cs

```diff
@@ -147,6 +148,27 @@ public void SendPingWithIPAddress(AddressFamily addressFamily)
                 });
         }
 
+        [Theory]
```

Comment:
again please lets minimize or avoid new Ping tests if we can by mixing in TimeSpan into the existing tests, so long as every overload gets called somewhere.